compose.yaml

```diff
@@ -96,14 +97,14 @@ services:
       - -exc
       - |
         echo "Installing curl"
-        
+
         apt-get update
         env DEBIAN_FRONTEND=noninteractive apt-get install curl -y
-        
+
         echo "Importing dump"
-        
-        curl --retry 50 --retry-max-time 3600 -sSL https://serve-trustify-dumps.apps.cluster.trustification.rocks/20241208T001823Z/dump.sql.gz -o dump.sql.gz
-        zcat dump.sql.gz | psql -v ON_ERROR_STOP=1
+
+        # curl --retry 50 --retry-max-time 3600 -sSL https://serve-trustify-dumps.apps.cluster.trustification.rocks/20241208T001823Z/dump.sql.gz -o dump.sql.gz
+        zcat /home/heliofrota/Desktop/dump.sql.gz | psql -v ON_ERROR_STOP=1
```

Comment:
That doesn't look right.

Reply:
:facepalm: I committed my local dump (aka THEDUMP.sql)
I'll update soon